drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
```

Comment:
at the moment we done even distinguish between rc and service names. all are named the same. so followed the existing convention. If we want to use `drupal_secret` for secrect, then maybe we should also think about `drupal_rc` and `drupal_svc`?